src/db_ledger.rs

```diff
@@ -614,19 +620,19 @@ mod tests {
     #[test]
     fn test_get_blobs_bytes() {
         let shared_blobs = make_tiny_test_entries(10).to_blobs();
+        let slot = DEFAULT_SLOT_HEIGHT;
         index_blobs(
-            shared_blobs.iter().zip(vec![0u64; 10].into_iter()),
+            shared_blobs.iter().zip(vec![slot; 10].into_iter()),
```

Comment:
next step is to get rid of this gross tuple thing?  broadcast should know its slot...